src/iGameStrings_cat.c

```diff
@@ -51,8 +51,13 @@
 
 #include <exec/memory.h>
 #include <libraries/locale.h>
+#include <libraries/iffparse.h>
 
+#include <proto/exec.h>
+#include <proto/dos.h>
 #include <proto/locale.h>
+#include <proto/utility.h>
+#include <proto/iffparse.h>
```

Comment:
Can you please help me understand why you added the above includes, since the changes you did at the rest of the code do not add new features, except new strings?

Reply:
Good question; this is from @T0lk13n's own work and the only thing I can think of is regenerating iGameStrings_cat.c from the original .cd file.

Reply:
Ups...Im really sure I didnt need it. I dont even know why should it be there.

Reply:
I'll clean these out after work today.